src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,56 @@
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+
+    padding: 0;
+    margin: 0;
+
+    list-style: none;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    position: relative;
+
+    display: flex;
+    align-items: center;
+
+    height: 60px;
+
+    font-weight: 500;
+    font-size: 12px;
+    color: $colorMain;
+    text-transform: uppercase;
+    text-decoration: none;
+    white-space: nowrap;
+    cursor: pointer;
```

Comment:
I don't understand why you use "cursor" here.  links have "cursor : pointer" by default.